deploy/charts/cert-manager/README.md

```diff
@@ -77,6 +77,7 @@ The following table lists the configurable parameters of the cert-manager chart
 | `global.rbac.create` | If `true`, create and use RBAC resources (includes sub-charts) | `true` |
 | `global.priorityClassName`| Priority class name for cert-manager and webhook pods | `""` |
 | `global.podSecurityPolicy.enabled` | If `true`, create and use PodSecurityPolicy (includes sub-charts) | `false` |
+| `global.podSecurityPolicy.useApparmor` | If `true`, use Apparmor seccomp profile in PSP | `true` |
```

Comment:
Can you make this `useAppArmor` (capatalised A on Armor)? This is in-line with the grafana chart too 😄

Reply:
Sure.